frontend/src/styles/_uswds-theme.scss

```diff
@@ -19,6 +19,13 @@ in the form $setting: value,
   $utilities-use-important: true,
 
   // Prettier defaults for typography
-  $theme-font-type-sans: "public-sans",
-  $theme-global-link-styles: true
+  $theme-font-type-sans: "source-sans-pro",
+  $theme-global-link-styles: true,
+
+  $background-color-palettes: (
+    "palette-color-system-cyan-light",
+    "palette-color-system-yellow-light",
+    "palette-color-system-green-cool",
+    "palette-color-system-red-warm"
+  )
```

Comment:
Nice! I figured there was a quick/easy way to make these classes available. Just checking, this appends to the available bg colors and doesn't overwrite them?

Reply:
Yup! Existing bg colors are still there.